LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Keep this back in the repo

Reply:
whoops! didnt realize it was missing, adding back in and creating a gitignore now.